092-self-healing-using-cruise-control.md

```diff
@@ -76,17 +79,18 @@ statusUpdateDate=2024-12-09T09:49:47Z,
 failedDisksByTimeMs={2={/var/lib/kafka/data-1/kafka-log2=1733737786603}}, 
 status=FIX_STARTED}]}
 ```
+where the `anomalyID` represents the task id assigned to the anomaly, `failedDisksByTimeMs` represents the disk which failed, `detectionDate` and `statusUpdateDate` represet the date and time when the anomaly was detected and the status of the anomaly was updated.
 
 The anomaly status would change based on how the healing progresses. The anomaly can transition to these possible status:
 * DETECTED - The anomaly is just detected and no action is yet taken on it 
 * IGNORED - Self healing is either disabled or the anomaly is unfixable
-* FIX_STARTED - The anomaly has started getting fixed
+* FIX_STARTED - The anomaly fix has started
 * FIX_FAILED_TO_START - The proposal generation for the anomaly fix failed 
 * CHECK_WITH_DELAY - The anomaly fix is delayed 
-* LOAD_MONITOR_NOT_READY - Monitor which is monitoring the workload of a Kafka cluster is in loading or bootstrap state
+* LOAD_MONITOR_NOT_READY - Monitor for workload of a Kafka cluster is in loading or bootstrap state
 * COMPLETENESS_NOT_READY - Completeness not met for some goals. The completeness describes the confidence level of the data in the metric sample aggregator. If this is low then some Goal classes may refuse to produce proposals.
 
-We can also poll the `state` endpoint, with `substate` parameter as `executor`, to get details on the tasks that are currently being performed by Cruise Control.
+We can again poll the [`state` endpoint](https://github.com/linkedin/cruise-control/wiki/REST-APIs#query-the-state-of-cruise-control), with `substate` parameter as `executor`, to get details on the tasks that are currently being performed by Cruise Control.
```

Comment:
Will we provide, how users might poll this end point?

Reply:
By this you mean mentioning the complete curl command?

Reply:
I have added the curl command to poll the endpoint